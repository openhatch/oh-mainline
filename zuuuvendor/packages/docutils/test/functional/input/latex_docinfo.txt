:Author: Foo Fred
:Organization: Food Foomatics & Friends
:Contact: foo@food.example.info
:Address: Fox St 13
          Foowood
:Author: Bar Barney
:Organization: Bar-BQ Bar
:Contact: 1-800-BARBQBAR
:Address: Barbara St 16
          South Barwell
