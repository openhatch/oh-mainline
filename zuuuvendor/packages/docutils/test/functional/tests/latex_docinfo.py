# Source and destination file names.
test_source = "latex_docinfo.txt"
test_destination = "latex_docinfo.tex"

# Keyword parameters passed to publish_file.
reader_name = "standalone"
parser_name = "rst"
writer_name = "latex"

# Extra setting we need

settings_overrides['use_latex_docinfo'] = 1
