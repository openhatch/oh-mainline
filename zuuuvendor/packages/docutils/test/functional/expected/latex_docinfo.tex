\documentclass[a4paper]{article}
% generated by Docutils <http://docutils.sourceforge.net/>
\usepackage{fixltx2e} % LaTeX patches, \textsubscript
\usepackage{cmap} % fix search and cut-and-paste in Acrobat
\usepackage{ifthen}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}

%%% Custom LaTeX preamble
% PDF Standard Fonts
\usepackage{mathptmx} % Times
\usepackage[scaled=.90]{helvet}
\usepackage{courier}

%%% User specified packages and stylesheets

%%% Fallback definitions for Docutils-specific commands

% hyperlinks:
\ifthenelse{\isundefined{\hypersetup}}{
  \usepackage[colorlinks=true,linkcolor=blue,urlcolor=blue]{hyperref}
  \urlstyle{same} % normal text font (alternatives: tt, rm, sf)
}{}
\hypersetup{
  pdfauthor={Foo Fred;Bar Barney}
}

%%% Title Data
\title{}
\author{Foo Fred\\
Food Foomatics \& Friends\\
foo@food.example.info\\
Fox St 13\\
Foowood \and
Bar Barney\\
Bar-BQ Bar\\
1-800-BARBQBAR\\
Barbara St 16\\
South Barwell}
\date{}

%%% Body
\begin{document}
\maketitle

\end{document}
